services-discovery/src/main/java/io/scalecube/services/discovery/ScalecubeServiceDiscovery.java

```diff
@@ -103,7 +105,10 @@ public ServiceEndpoint endpoint() {
   private ClusterConfig.Builder clusterConfigBuilder(ServiceDiscoveryConfig config) {
     Builder builder = ClusterConfig.builder();
     if (config.seeds() != null) {
-      builder.seedMembers(config.seeds());
+      builder.seedMembers(
+          Arrays.stream(config.seeds())
+              .map(address -> Address.create(address.host(), address.port()))
```

Comment:
Would u mind if I ask u to fully specify Address type here. Just to be in sync with line 48